lib/fog/aws/requests/storage/post_object_restore.rb

```diff
@@ -15,11 +16,16 @@ class Real
         #
         # @see http://docs.amazonwebservices.com/AmazonS3/latest/API/RESTObjectPOSTrestore.html
         #
-        def post_object_restore(bucket_name, object_name, days = 100000)
+        def post_object_restore(bucket_name, object_name, days = 100000, tier = nil)
```

Comment:
Rather than adding another argument here, I think it might be a good idea to covert this to using an options hash (which could then include both days and tier, but also allow for future compatibility with additional options).

You can see an example of how we've done that conversion on other methods here: https://github.com/fog/fog-aws/blob/c84f6aa666a2626bca308082510f31d3e7e6b0b8/lib/fog/aws/requests/compute/create_volume.rb#L32-L36

Does that make sense? Otherwise everything here looks good, but I think going directly to that style will save us some trouble in the long run.